plugins/GrapesJsBuilderBundle/Assets/library/js/builder.service.js

```diff
@@ -260,15 +260,16 @@ export default class BuilderService {
    */
   getAssetManagerConf() {
     return {
-      assets: this.assets,
-      noAssets: Mautic.translate('grapesjsbuilder.assetManager.noAssets'),
-      upload: this.uploadPath,
```

Comment:
@kuzmany I guess we can then remove this parameter? And I think the actual PHP endpoint too.

Reply:
Yes, you're true